content/en/docs/faq/multi-cluster-management/manage-multi-cluster.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Manage a Multi-cluster Environment on KubeSphere"
+keywords: 'Kubernetes, KubeSphere, federation, multicluster, hybrid-cloud'
+description: 'Understand how to manage a multi-cluster environment on KubeSphere.'
+linkTitle: "Manage a Multi-cluster Environment on KubeSphere"
+weight: 16710
+---
+
+KubeSphere provides an easy-to-use multi-cluster feature to help you [build your multi-cluster environment on KubeSphere](../../../multicluster-management/). This guide illustrates how to manage a multi-cluster environment on KubeSphere.
+
+## Prerequisites
+
+- Make sure your Kubernetes clusters are installed with KubeSphere before you use them as your Host Cluster and Member Clusters. 
+
+- Make sure the cluster role is set correctly on your Host Cluster and Member Clusters respectively, and the `jwtSecret` is the same between them.
+
+- It is recommended that your Member Cluster is a clean environment where no resources have been created on it before it is imported to the Host Cluster.
+
+
+## Manage your KubeSphere Multi-cluster Environment
+
+Once you build a multi-cluster environment on KubeSphere, you can manage it through the central control plane from your Host Cluster as you can select a specific cluster when creating resources. You need to avoid creating resources on your Host Cluster to prevent it from overloading. It is not recommended to log in to the KubeSphere web console of your Member Clusters to create resources on them as these resources won't be synchronized to your Host Cluster for management.
+
+### Resource Management
+
+It is recommended that you don't change a current Host Cluster to a Member Cluster or the other way round. If a Member Cluster has been imported to a Host Cluster before, you have to use the same cluster name when unbinding and importing it to a new Host Cluster.
+
+If you want to import the Member Cluster to a new Host Cluster while retaining existing projects (i.e. namespaces), you can follow the steps as below.
+
+1. Run the following command on the Member Cluster to unbind the projects to be retained from your workspace.
+
+   ```bash
+   kubectl label ns <namespace> kubesphere.io/workspace- && kubectl patch ns <namespace>   -p '{"metadata":{"ownerReferences":[]}}' --type=merge
+   ```
+
+2. Run the following command on the Member Cluster to clear your workspace.
+
+   ```bash
+   kubectl delete workspacetemplate <workspace name>
+   ```
+
+3. When you create a workspace on the new Host Cluster and authorize the Member Cluster to this workspace, run the following command on the Member Cluster to bind the projects retained to the workspace.
```

Comment:
authorize the Member Cluster to
->
assign the Member Cluster to

You can say "authorize sth" or "authorize sb to do" but I am actually wondering if it is correct to say "authorize sth to sth/sb". If you find any reference, tell me.

Reply:
retained to the workspace.
->
retained for the workspace.